ipynb/android/Android_Workloads.ipynb

```diff
@@ -316,7 +316,7 @@
    },
    "outputs": [],
    "source": [
-    "SYSTRACE_CMD  = CATAPULT_HOME + \"/systrace/systrace/systrace.py -e {} -o {} gfx view sched freq idle -t {}\"\n",
+    "SYSTRACE_CMD  = CATAPULT_HOME + \"/systrace/systrace/run_systrace.py -e {} -o {} gfx view sched freq idle -t {}\"\n",
```

Comment:
We could try to consolidate this change by adding a new Android::System API to start systrace and wait for its completion. This allows also to solve the issue reported by @bjackman since we can lookup for the proper binary under CATAPULT_HOME.

Something like:

```python
class System(object):
    """
    Collection of Android related services
    """

    @staticmethod
    def systrace_start(target, trace_file, time, events=[gfx view sched freq idle])):
       # Check which systreace binary is available under CATAPULT_HOME

       #  Format the command according to the specified time and events

       # Actually spawn systrace using code similar to what we have in the notebooks, i.e.
       trace_cmd = SYSTRACE_CMD.format(target.device, trace_file, time)
       logging.info('SysTrace: %s', trace_cmd)
       systrace_output = pe.spawn(trace_cmd)
       return systrace_output

    @staticmethod
    def systrace_wait(target, systrace):
        systrace_output.wait()
```